typescript/cli/cli.ts

```diff
@@ -73,14 +70,7 @@ try {
     .demandCommand()
     .strict()
     .help()
-    .fail((msg, err, yargs) => {
-      if (msg && !msg.includes(MISSING_PARAMS_ERROR)) errorRed('Error: ' + msg);
-      console.log('');
-      yargs.showHelp();
-      console.log('');
-      if (err) errorRed(err.toString());
```

Comment:
Do we definitely want to cut this line? Can a command fail without throwing an error?

Reply:
https://yargs.js.org/docs/#api-reference-failfn-boolean
https://yargs.js.org/docs/#api-reference-showhelponfailenable-message
customizing the fail handler seems unnecessary and I dont really understand the conditions in here

Reply:
atm the help string is logged whether or not the error is related to a missing argument